values.yaml

```diff
@@ -104,3 +104,52 @@ oda:
   use_hostpath: true
   ddaQueue: queue-osa11
   ddcache: "/net/cdcicn02/scratch/shared/savchenk/data/reduced/ddcache"
+
+plugins:
+  osa:
+    ebabled: true
+    instruments:
+      isgri:
+        dispatcher_mnt_point: /data
+        data_server_cache: reduced/ddcache
+        dummy_cache: /data/dummy_prods
+        data_server_url: http://oda-dda-interface:8000
+
+      jemx:
+        dispatcher_mnt_point: /data
+        data_server_cache: reduced/ddcache
+        dummy_cache: /data/dummy_prods
+        data_server_url: http://oda-dda-interface:8000
+
+  magic:
+    enabled: true
+    instruments:
+      magic:
+        data_server_url: http://oda-magic:8000
+        dummy_cache: /data/dummy_prods
+
+  polar:
+    enabled: true
+    instruments:
+      polar:
+        dispatcher_mnt_point:
+        data_server_cache:
+        dummy_cache: /data/dummy_prods
+        data_server_url: polar-worker
+        data_server_port: 8893
+
+  antares:
+    enabled: true
+    instruments:
+      antares:
+        data_server_url: http://oda-antares:8000
```

Comment:
when it is deployed along with the dispatcher, it will be fixed, you are right. I just imagined the situation when backend is separate, like in case of spi.

Reply:
indeed! that's why there below I pointed out that special case. That one option could be configurable. 
It could be better if it is configurable with combined configs, and given as example.

Really, it is also possible that other backends are detached. E.g. we did run dda one rather separately. 
So if general method is adopted could be nicer. But it's effort to make it consistent throughout.

Non-general method would be to just get fields from values, those that are assumed to be modifiable.

Reply:
by the way, small detail, no way for you to know, but SPI is an entirely different case than SPI-ACS : different science, different data reduction, software written in different places. They were even built in different countries. If you mention you have SPI analysis to some colleagues in France they might get quite confused.